certs/certs.c

```diff
@@ -77,4 +80,6 @@ const char certificates[] =
 "PIWmawomDeCTmGCufsYkl4phX5GOZpIJhzbNi5stPvZR1FDUWSi9g/LMKHtThm3Y\r\n"
 "Johw1+qRzT65ysCQblrGXnRl11z+o+I=\r\n"
 "-----END CERTIFICATE-----\r\n"
+#endif
```

Comment:
usually good practice to end the `#endif` with a comment describing what it's closing. ex:

`#endif  // LEAN_CERTS

Reply:
Done.